### ISSUE TITLE
Unable to transfer MySQL dashboards between computers.

### ISSUE THREAD

**Issue by atharvaa1337** [grafana/grafana#54136]
**What happened**:
I have two databases which are identical, and both are independently stored on two separate computers. If I make a grafana UI for monitoring one of the databases, it works fine on that computer. However when I export it to a JSON and send it to my second computer, all I get is a collection of blank boxes with no data on the second computer, despite the databases between the computers being the exact same.

**What you expected to happen**:
I expected the same dashboard JSON to work the same on two separate computers.

**How to reproduce it (as minimally and precisely as possible)**:
Have two identical SQL databases on two computers. Make a grafana UI on one computer, turn it into a SQL, send it to the second computer. The second computer only opens blank boxes. 

**Anything else we need to know?**:
I have tried replacing all the Unique identifiers and ensured grafana version and SQL version is the same between the two computers. 

**Environment**:
- Grafana version: 9.1.0 (both systems)
- Data source type & version: MySQL Workbench CE 8.0.30 (both systems)
- OS Grafana is installed on: Windows 11 (both systems)
- User OS & Browser: (both systems) Chrome Version 104.0.5112.102 (Official Build) (64-bit)
- Grafana plugins: none?
- Others: If this is not a bug and the fault is mine, I apologize. Please close this issue and let me know to fix this.

**Comment by gelicia** [comment]
@atharvaa1337 If you have corrected the unique identifiers for datasources on the second instance where the panels are not working, and you go in and edit those panels, does it provide any error messages on why the query isnt running?

**Comment by atharvaa1337** [comment]
@gelicia Thank you so much, I am sorry to create this as a issue when the fault was mine. I did a single change in the panel and saw the Unique ID change for the JSON just for that panel. I then did a find and replace for all the old panels with the new one unique identifier and all the panels began working magically. I just wish grafana would autogenerate the unique ID for a particular DB as I had no clue where to even find mine in the beginning. That being said, thanks for this otherwise wonderful software and have a great day!
